src/jobflow/managers/fireworks.py

```diff
@@ -153,6 +153,11 @@ def run_task(self, fw_spec):
             store = SETTINGS.JOB_STORE
         store.connect()
 
+        # Add the metadata from the fw_spec
+        fw_tags = fw_spec.get("tags", None)
+        if fw_tags is not None:
+            job.metadata.update({"tags": fw_tags})
```

Comment:
Maybe we should also check if tags already exist and if they are a list, and in that case append the tags?

Reply:
Alright. Added.

One small issue I encountered here was that tags were getting duplicated if the job metadata had the "tags" key in it. Because of this line:  [ spec.update(job.metadata)  # add metadata to spec](https://github.com/materialsproject/jobflow/blob/1386b113e18fb0faf42992d1e2ed3a379b206123/src/jobflow/managers/fireworks.py#L116). 

To account for this, I just removed all duplicates from `job.metadata["tags"]` assuming that duplicates will never be desired.

Reply:
That's a good catch. Thanks!